app/routers/agenda.py

```diff
@@ -39,6 +40,21 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    get_a_week_int = 6
+    start_date_graph, end_date_graph = calc_dates_range_for_agenda(
+        0, 0, get_a_week_int)
```

Comment:
Consider changing to keyword arguments to make it more readable

Reply:
done